### DISCUSSION TITLE
关于巴以冲突的一些资料搜集和看法

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#23] [Announcements]
# 关于巴以冲突的一些资料搜集和看法

帝国主义的余毒。

https://zexwoo.blog/posts/editorial/palestine-vs-israel/

**Comment by lightleesunny** [comment]
世界就是存量市场，中国的崛起必然一些不思进取的国家倒下。本来就是强盗出身的他，会任人宰割吗？

**Comment by williamszw** [comment]
在以色列占优势的情况下，在“集中营”里的巴勒斯坦阿拉伯人的人口翻番了。在阿拉伯人占优势的情况下，以色列可能早就不存在了。谁更文明？
